mismi-s3/test/Test/IO/Mismi/S3/Commands.hs

```diff
@@ -151,6 +151,17 @@ prop_upload d l = withLocalAWS $ \p a -> do
   r <- read a
   pure $ r === Just d
 
+prop_s3Sink :: T.Text -> Property
```

Comment:
I think something like `sourceList` from [Conduit.Data.List](https://hackage.haskell.org/package/conduit-1.2.4.2/docs/Data-Conduit-List.html) would be useful here. Don't gain anything from writing/read before you sink, could just as easily create some source and sink it.